docs/testing-pre-releases.md

```diff
@@ -96,16 +96,16 @@ yum install <package name>-<version number>
 
 ### Getting kubeadm binaries from a GCS bucket
 
-Pre-compiled GA, alpha/beta versions of kubeadm binary are deployed into `gs://kubernetes-release/release/` GCS bucket,
+Pre-compiled GA, alpha/beta versions of kubeadm binary are deployed into `https://dl.k8s.io/release/` GCS bucket,
 while CI/CD versions are deployed into `gs://k8s-release-dev/ci/` bucket.
 
-To explore versions available in Google Storage buckets use:
+To explore versions available in Storage buckets use:
 
 ```bash
 gsutil ls gs://{bucket-name}{filter}
```

Comment:
we are leaving some gs:// urls and example commands in the repo. is this intentional?

Reply:
The reason behind the intent of https://github.com/kubernetes/k8s.io/issues/2396 is to shift away from relying solely on GCP resources. Given this objective, I believe the change made in this PR is appropriate and deliberate. Although not all resources are currently prepared to transition away from using `gs://`, the introduction of `dl.k8s.io` will enable the utilization of alternative cloud providers.

Reply:
That's fair, I'll figure out something to make them a bit more clear. A similar change was added below referencing the binary copy. https://github.com/kubernetes/kubeadm/pull/2872/files#diff-8902b8f116e729ad637246192befaca7fc3954d1226eb8368b51d7261880136eR116-R121

I worry that a direct replacement like `curl -X GET "https://storage.googleapis.com/storage/v1/b/kubernetes-release/o?delimiter=/&prefix=release/{filter}"` is a bit convoluted. 

On second thought, maybe for these specific doc references it's best to roll back to still using `gsutil` since AFAIK `dl.k8s.io` doesn't have a way to list resources via an http request like `gsutil` is doing here. @BenTheElder do you have any recommendations?

Reply:
We should stop depending on the fact that it's backed by a GCS bucket, it could be something else in the future.

If it's the release-dev bucket you can depend on it being GCS, that's not end-user facing.

For kubernetes releases, they should be finding releases by using the release notes, where they're already advertised as dl.k8s.io URLs.